core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -407,6 +431,14 @@ public long getUsedBytes() {
     if (mSafeModeManager.isInSafeMode()) {
       throw new UnavailableException(ExceptionMessage.MASTER_IN_SAFEMODE.getMessage());
     }
+    try {
+      return mWorkerInfoCache.get(WORKER_INFO_CACHE_KEY);
+    } catch (ExecutionException e) {
+      throw new UnavailableException("Unable to get worker info list from cache");
```

Comment:
Can we include the information from `e` in this exception message?

Reply:
Yeah, added the exception message